scripts/install-binary.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+
+'use strict'
+
+var which = require('which')
+var wget = require('download')
+var fs = require('fs')
+var path = require('path')
+
+var JQ_NAME = 'jq'
+
+var JQ_RELEASE_INFO = {
+  url: 'http://github.com/stedolan/jq/releases/download/',
+  version: 'jq-1.5'
+}
+
+var DOWNLOAD_URL = JQ_RELEASE_INFO.url + JQ_RELEASE_INFO.version + '/jq-'
+var ROOT_PATH = path.join(__dirname, '..')
+var BINARY_PATH = path.join(ROOT_PATH, 'node_modules/.bin/')
+var SYSTEM_PATH = process.env.PATH
+
+/**
+ * Returns a clean path that helps avoid `which` finding bin files installed
+ * by NPM for this repo.
+ * @param {string} path
+ * @return {string}
+ */
+
+function cleanPath (path) {
+  return path
+      .replace(/:[^:]*node_modules[^:]*/g, '')
+      .replace(/(^|:)\.\/bin(:|$)/g, ':')
+      .replace(/^:+/, '')
+      .replace(/:+$/, '')
+}
+
+function getBinaryLocation (binary) {
+  try {
+    return which.sync(binary)
+  } catch (err) {
+    console.log('Error: ', err)
+    process.exit(1)
+  }
+}
+
+function isInstalledGlobally (binary) {
+  return !!getBinaryLocation(binary)
```

Comment:
I mean, the boolean cast is still there. If you really want it to be a boolean, do `Boolean(getBinaryLocation(binary))`, not the `!!`.

I prefer it to actually have a proper comparison though. 

Right now, the try/catch didn't add anything useful. The exception `getNotFoundError` is thrown when it can't find the binary, so we should catch that exception so we can recover from it, and then return an empty string, for example. So it would look like:

```javascript
function getBinaryLocation (binary) {
  try {
    return which.sync(binary)
  } catch (err) {
    if (err instanceof getNotFoundError) {
      return ''
    } else {
      throw err
    }
  }
}

function isInstalledGlobally (binary) {
  return getBinaryLocation(binary) !== ''
}
```